lib/lunchbot_web/live/user_auth_live.ex

```diff
@@ -0,0 +1,19 @@
+defmodule LunchbotWeb.UserAuthLive do
+  import Phoenix.LiveView
+  alias Lunchbot.Accounts
+
+  # TODO:
+  #   https://blog.appsignal.com/2022/01/25/securing-your-phoenix-liveview-apps.html
+  #   & https://stackoverflow.com/questions/66217782/elixir-phoenix-liveview-passing-user-id-through-socket
+  def on_mount(:user, _params, %{"user_token" => user_token} = _session, socket) do
+    socket =
+      socket
```

Comment:
could you just do `socket |> assign(:current_user, Accounts.get_user_by_session_token(user_token))` here?

Reply:
No because then `socket` will never be redefined and in the lines down below we will get an error for `socket.assigns.current_user` not existing